src/routes/feedback/components/FeedbackBox.js

```diff
@@ -132,9 +136,9 @@ export class FeedbackBox extends React.Component<PropsType, StateType> {
     const currentRoute = location.type
 
     const options = []
-    const currentPageFeedbackLabel = this.getCurrentPageFeedbackLabel()
-    if (currentPageFeedbackLabel) {
-      options.push(this.getFeedbackOption(currentPageFeedbackLabel, this.getCurrentPageFeedbackType()))
+    const currentPageFeedbackOption = this.getCurrentPageFeedbackOption()
+    if (currentPageFeedbackOption) {
+      options.push(currentPageFeedbackOption)
```

Comment:
I think this can be done shorter with 
```
const options = this.getCurrentPageFeedbackOption() ? [this.getCurrentPageFeedbackOption()] : []
```
right?